### DISCUSSION TITLE
There was an error but I didn't find any more information on the terminal and the training kept running.

### DISCUSSION THREAD

**Discussion by sssua-sx** [talmolab/sleap#1186] [General]
hi.
when i clicked  [**run training** ] and  set up some  parameter to click [**run**] , i saw   a new interface. But it pops up very quickly  that "An error occurred while training centroid. Your command lineterminal may have more information about the error",and the interface disappeared. I didn't find an error in the terminal and the terminal kept running until I saw  that "Finished training centered_instance." I don't know what I'm supposed to do and I don't see any changes in the training video.   

 thanks!
![7a8b28592133fe54e0562e441ba545c](https://user-images.githubusercontent.com/58169347/220936704-c059c7de-003e-4367-8d65-d36b3cd51d79.png)
![image](https://user-images.githubusercontent.com/58169347/220936836-e96a688e-c2f3-4721-a358-2cbd7e4c3a53.png)

**Comment by roomrys** [comment]
Hi @sssua-sx,

Even if an error was encountered while training one model, SLEAP will attempt to train the second model. The error message only pops-up after you close the dialog, but(!) as soon as you close the dialog, the second model will start training - making the error message difficult to locate...

Could you post your terminal output starting from `Happy SLEAPing!`?

Thanks,
Liezl

**Reply by sssua-sx** [reply]
Thank you very much. I found an error in the happy sleaping terminal using the method you provided. I guessed it was caused by my gpu not enough. Now I am trying to modify the size of the picture or batch-size, hoping to succeed. Thank you again for your answers.

**Comment by roomrys** [comment]
Hi @sssua-sx,

I wanted to update you and say that I experienced the same issue and found that it is due to a `proc.returncode` of `1` even though the operation was seemingly successful (i.e., checking the predictions in the `Saved predictions: ...` file, the predictions look good). 
https://github.com/talmolab/sleap/blob/1c75ab82f4f08fcaa42b5de54a43ad169b296116/sleap/gui/learning/runners.py#L853-L860

I am looking into this issue to see what may have changed to cause this, although I have an inkling it is [this line](https://github.com/talmolab/sleap/blame/develop/sleap/nn/training.py#L2009).

Thanks,
Liezl
